src/rsvc/srv_common.c

```diff
@@ -238,15 +237,25 @@ ds_rsvc_get_attr(struct ds_rsvc *svc, struct rdb_tx *tx, rdb_path_t *path,
 
 		rc = rdb_tx_lookup(tx, path, &key, &iovs[j]);
 
-		if (rc != 0) {
-			D_CDEBUG(rc == -DER_NONEXIST, DB_ANY, DLOG_ERR,
-				 "%s: failed to lookup attribute "DF_KEY
-				 ": "DF_RC"\n",
-				 svc->s_name, DP_KEY(&key), DP_RC(rc));
+		if (rc == -DER_NONEXIST) {
+			/* attribute do not exist */
+			iovs[j].iov_buf_len = sizes[i];
+			iovs[j].iov_len = 0;
+			sizes[i] = 0;
+			/* fake crt_sgl_valid() */
+			iovs[j].iov_buf = (void *)(-1);
+
+			D_DEBUG(DB_ANY, "%s: failed to lookup attribute "DF_KEY": "DF_RC"\n",
```

Comment:
shall we change this message from failed to non-existent?
or maybe better just remove it